table_one.py

```
@@ -413,29 +432,77 @@ def generate_table(
+        df, var_meta or {}, strategy='complete-case', return_counts=True
+    )
+    
+    # Track missing data info for report
+    missing_data_info = {
+        'strategy': 'complete-case',
+        'rows_analyzed': miss_counts['final_rows'],
+        'rows_excluded': miss_counts['rows_removed'],
+        'summary_before': missing_summary_records
+    }
+    
+    logger.info(f"Missing data: {miss_counts['rows_removed']} rows excluded ({miss_counts['pct_removed']:.1f}%)")
```

Ruff (0.14.13)

Logging statement uses f-string

(G004)